data/core/units/orcs/Warrior.cfg

```diff
@@ -14,7 +14,7 @@
     advances_to=Orcish Warlord
     cost=26
     usage=fighter
-    description= _ "Having mastered the art of dual-sword fighting, Orcish Warriors are highly respected among the Orcs, and highly feared by their foes. Their only weakness is their inability to wield the bow."
+    description= _ "Orcish Warriors are those orcs who have proved their strength on the battlefield, choosing to display their skill by fighting with a sword in each hand. Their prowess in battle has gained them the respect of other orcs, and they will typically be seen ordering lesser bands of raiders. Despite their position, they rarely possess the commanding skills held by the leaders of other armies."
```

Comment:
.. and they **typically will** be seen ..

or

.. and**, typically,** they will be seen ..